query-service-impl/src/test/resources/prometheus_request_handler.conf

```diff
@@ -0,0 +1,42 @@
+{
+  name = raw-service-prometheus-handler
```

Comment:
Can we go for individual scope level say for service and API separate handler? e.g `raw-service-prometheus-handler-service`